src/components/ProjectDashboard/components/PayProjectModal/PayProjectModal.tsx

```diff
@@ -139,45 +138,18 @@ export const PayProjectModal: React.FC = () => {
                         )}
                       >
                         <Trans>
-                          I accept the{' '}
+                          I understand and accept
```

Comment:
This might have unintended issues with translations - could we split it into two separate strings?

Reply:
not that it matters too much, but for future ref: strings marked for translation should always be whole sentences. 

made the change here:
cc @filipviz

Reply:
Thank you!